frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -307,7 +310,10 @@ export const WhatsNewMenu = (props: Props) => {
     },
   };
   // Only show its announcement if tracker removal is live:
-  if (typeof props.profile.remove_level_one_email_trackers === "boolean") {
+  if (
+    typeof props.profile.remove_level_one_email_trackers === "boolean" &&
```

Comment:
```suggestion
```
We do not need to check the profile to see if user has enabled remove email tracker since this is news.

Reply:
Updated.

Reply:
This wasn't a check to see whether the user has it enabled (it doesn't check for true or false), but just if the back-end provides the value in general (i.e. whether it's of type `boolean` rather than `undefined`) :slightly_smiling_face: